src/components/InstanceMonitoring/_instance-monitoring.scss

```diff
@@ -0,0 +1,51 @@
+@import 'carbon-components/scss/globals/scss/colors';
+@import 'carbon-components/scss/globals/scss/typography';
+@import 'carbon-components/scss/globals/scss/vars';
+
+.instance-card {
+  background-color: #fff;
+  width: 1167px;
+  height: 210px;
+  .instance-metrics {
+    display: flex;
+    justify-content: center;
+    li {
+      padding: 20px;
+      margin: 20px;
+      text-align: center;
+      font-size: 25px;
+      font-weight: bold;
+      .instance-amount {
+        font-size: 36px;
+        padding-right: 5px;
+      }
+      .instance-unit {
+        font-size: 14px;
+      }
+      .instance-desc {
+        font-size: 16px;
+      }
+    }
+  }
+  .instance-hide {
+    display: flex;
+    margin-top: 20px;
+    hr {
+      width: 100%;
+      margin: auto;
+      border-top: none;
+      border-bottom: 2px solid #dfe3e6;
+    }
+    .instance-hide-link {
+      color: #bcc8d1;
+      width: 30%;
+      text-align: center;
+      font-weight: bold;
```

Comment:
Carbon codebase uses `400`/`500`/`600`/`700` for `font-weight`, so good to do the same here.